web/angular-wallet/src/app/main/messages/messages.service.ts

```diff
@@ -159,26 +147,6 @@ export class MessagesService implements Resolve<any> {
     return transactionId;
   }
 
-  /**
-   * Select options
-   *
-   * @param options
-   */
-  selectOptions(options): void {
-    this.onOptionsSelected.next(options);
-  }
-
-  /**
-   * Get contacts
-   *
-   * @returns {Promise<any>}
-   */
-  getContacts(): Promise<any> {
-    return new Promise((resolve, reject) => {
-      resolve([]);
-    });
```

Comment:
ok. will check this

Reply:
even weirder as my search did not reveal the usage....wtf?
![image](https://user-images.githubusercontent.com/3920663/57394913-17866a80-719d-11e9-8c34-5c1cf8b45908.png)

Reply:
well, the `contacts` array is never being used!

Reply:
https://github.com/burst-apps-team/phoenix/blob/develop/web/angular-wallet/src/app/main/messages/sidenavs/left/message/message.component.html#L51

Reply:
but read-only...or, am I blind?

Reply:
Just wanna make sure we didnt break it, since now the template refers to an undefined var right?

Reply:
This is used here: https://github.com/burst-apps-team/phoenix/blob/develop/web/angular-wallet/src/app/main/messages/sidenavs/left/message/message.component.ts#L62